be/src/exec/pipeline/stream_epoch_manager.cpp

```diff
@@ -96,4 +133,14 @@ const BinlogOffset* StreamEpochManager::_get_epoch_unlock(const TabletId2BinlogO
     }
 }
 
-} // namespace starrocks
+size_t StreamEpochManager::activate_parked_driver(ExecEnv* exec_env, const TUniqueId& query_id) {
+    // TODO(lism): how to decide use driver_executor or wg_driver_executor?
+    size_t num_activated_drivers = 0;
+    num_activated_drivers += exec_env->driver_executor()->activate_parked_driver(
+            [query_id](const pipeline::PipelineDriver* driver) { return driver->query_ctx()->query_id() == query_id; });
+    num_activated_drivers += exec_env->wg_driver_executor()->activate_parked_driver(
```

Comment:
for one query, it must be executed by driver_executor() or wg_driver_executor(). why should active drivers of both here?

Reply:
fixed.